R/convert_to_na.R

```diff
@@ -125,7 +125,7 @@ convert_to_na.data.frame <- function(x, na = NULL, select = NULL, exclude = NULL
   .attach_packages(conflicting_packages)
 
   # return valid column names, based on pattern
-  select <- .evaluated_pattern_to_colnames(select, data, ignore_case, verbose, exclude)
+  select <- .evaluated_pattern_to_colnames(select, x, ignore_case, verbose, exclude)
```

Comment:
@IndrajeetPatil  @DominiqueMakowski Sometimes, we use `data`, sometimes `x`. Can we decide on a unique name for the data argument?

Reply:
If the function works primarily with dataframes, then `data`; if it works with vectors, then `x`.

If both, then it's tricky. But I guess `data` should be fine. It's generic enough to accommodate different data structures.